### PR TITLE
docs: update outdated country map tools instructions

### PR DESCRIPTION
**Pull request by ktmud**
### SUMMARY

Update the Country Map Tools doc to reflect [the new process](https://github.com/apache-superset/superset-ui/pull/1045) of adding a country map.

Also removed redundant information about ISO codes to make the docs easier to read (added a link to Wikipedia for anyone interested).

Also updated error handling in viz.py for country map to give more actionable errors.

#### Before

before

#### After

after

### TEST PLAN

CI and verify docs site locally

### CODE REVIEW

**Comment by ktmud** [comment]
For some reason I couldn't get the local doc builds to work.

```
        
success run static queries - 3.785s - 5/5 1.32/s
success run page queries - 0.073s - 125/125 1717.76/s
error Generating development JavaScript bundle failed

JSON5: invalid end of input at 1:1
Occurred while linting /Users/jesse_yang/projects/superset-fork/docs/src/pages/404.jsx:22
failed Building development bundle - 33.582s
success building schema - 3.975s
```

@pkdotson , have you seen this error before?

File: docs/src/pages/docs/Miscellaneous/index.mdx [hunk 1]
```diff
@@ -6,89 +6,57 @@ index: 1
...
-We can apply these concepts to specify the state of Texas in the country of United States:
-
-```
-US-TX
-```
-
-### Included Codes
-
-The ISO codes for the following countries are included in Superset:
+The Country Maps visualization already ships with the maps for following countries:
```
Comment (michael-s-molina) [inline docs/src/pages/docs/Miscellaneous/index.mdx:20]:
```suggestion
The Country Maps visualization already ships with the maps for the following countries:
```

File: docs/src/pages/docs/Miscellaneous/index.mdx [hunk 2]
```diff
@@ -6,89 +6,57 @@ index: 1
...
+- Portugal
 - Russia
 - Singapore
 - Spain
 - Switzerland
-- United Kingdom
+- Syria
+- Thailand
+- Timorleste
+- Uk
```
[the remark this answers is not in the document]
Comment (michael-s-molina) [reply docs/src/pages/docs/Miscellaneous/index.mdx:47]: I'm assuming that you're not talking about configuration values here, just listing the supported countries. If these are configuration values, can we change the text to clearly state that?

**Review by michael-s-molina — COMMENTED** [review]
Thanks for updating the docs! 🚀

**Review by michael-s-molina — APPROVED** [review]
LGTM

**Review by kristw — APPROVED** [review]

**Pull request merged**